README.md

```diff
@@ -181,16 +179,56 @@ console.log($);
 ## Python
 
 1. Python code SHOULD conform to the PEP8 and PEP257 guides
-2. Exceptions:
+1. Exceptions:
     1. 160 column limit
 
+## AWS
+
+### DNS Names
+
+DNS Records MUST adhere to the following pattern
+
+```text
+(instance).aws_service.(az).(region).service.environment.namespace
+```
+
+1. instance SHALL be used in the case that there are multiple instances of an aws_service, e.g. seperate databases.
```

Comment:
Oh wait. I see that they are the keywords. In which case they should remain in their current capitalisation but made `monospaced`.